samples/Chat/src/app/ChatArea.tsx

```diff
@@ -31,12 +31,16 @@ export const ChatArea = (props: ChatAreaProps): JSX.Element => {
 
   return (
     <Stack className={chatAreaContainerStyle}>
-      <MessageThread {...chatThreadProps} onRenderAvatar={props.onRenderAvatar} numberOfChatMessagesToReload={5} />
+      <MessageThread
+        {...(chatThreadProps as any)}
+        onRenderAvatar={props.onRenderAvatar}
+        numberOfChatMessagesToReload={5}
+      />
       <Stack.Item align="center" className={sendBoxParentStyle}>
         <div style={{ paddingLeft: '0.5rem', paddingRight: '0.5rem' }}>
-          <TypingIndicator {...typingIndicatorProps} />
+          <TypingIndicator {...(typingIndicatorProps as any)} />
```

Comment:
given Nan has a change for updating the usePropsFor, and we are migrating to using the composites instead of this raw usage, I'm inclined to not debug into this..

Reply:
however it is something that our end users might be hitting

Reply:
I will file a bug instead